### ISSUE TITLE
subject state attribute suggestions

### ISSUE THREAD

**Issue by AnnaHilverling** [HumanBrainProject/openMINDS_controlledTerms#118]
I would like to propose two suggestions for "Attributes" in "Subject state":

1) drug infusion (should represent recordings taken during the infusion of a drug).
2) drug treatment (should represent recordings taken while the drug has developed its final "stable" dose).

**Comment by UlrikeS91** [comment]
Thanks for the suggestions. To me, at least the second suggestion sound reasonable. The name would need to be different - something along the lines of "drug treated", "dosed" or "drugged" to be consistent with the naming of other subjectAttributes.

Could you give a rough description of the use case? "drug infusion" is an activity/process, so my first intuition would be to capture it as a protocol/protocolExecution where an undrugged subjectState is linked as input and the drugged subjectState as output. If the subject has been recorded in parallel, then the protocolExecution should reflect that and the recording file could be an additional output next to the drugged subjectState.
In short:
subjectState[undrugged] <--input-- protocolExecution[drug infusion + parallel recording] --output--> subjectState[drugged] + file[recorded data during drug infusion] 

The drugged subjectState could then be the input to another protocolExecution for the recording of the "stable" state. 

@lzehl what do you think?

**Comment by AnnaHilverling** [comment]
I see the point. I am fine with all your suggestions: "drugged", "drug treated", or "dosed".

The use case is that I have several recordings for one subject. These recordings were made once without drug treatment (control, 9 files)), once with drug treatment (9 files) and once while the drug was administered (drug infusion, 3 files)).
I would like to specify different subject states and set an attribute for more information.

**Comment by UlrikeS91** [comment]
@lzehl could you please give some feedback so that we can conclude on a solution for @AnnaHilverling.

**Comment by lzehl** [comment]
@AnnaHilverling thanks for raising this issue. @UlrikeS91 thanks for pushing me to provide feedback :smile: 

Adding the terms "untreated" and "drugged" as attributes for subjects should solve case to differentiate between undrugged and fully drugged subject states. 

The other case is indeed an interesting problem, since a recording happening while a subject is in the process of being drugged.

@AnnaHilverling I would solve this the following way (without the need of defining protocol executions for now):
subject: 
- state 1: untreated
- state 2: drugged; additional remarks "start of drug administration period"
- state 3: drugged; additional remarks "stable drug level period" (maybe you can even state the relative time to state 2)

@UlrikeS91 that would mean we need to add "drugged" and "untreated" to the SubjectAttributes.

**Comment by AnnaHilverling** [comment]
@lzehl and @UlrikeS91 thank you for bringing this topic up again :)

I will add this information to additional remarks. As soon as 'untreated' and 'drugged' are available, I will attach these attributes as well.

**Comment by lzehl** [comment]
solved with #175
